ansible/roles/apache-application-lavarel5/files/lavarel5.conf

```diff
@@ -0,0 +1,8 @@
+DocumentRoot ${baseRootDir}/public
+DirectoryIndex index.php
+
+<Directory ${baseRootDir}/public>
```

Comment:
I would only define the document root, every thing else gets delivered by .htaccess of the projects